recipes/mpfr/build.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+export LD_LIBRARY_PATH=$PREFIX/lib:$LD_LIBRARY_PATH
```

Comment:
Appears that `brew` may install its own copy of `gmp`, which was causing issue for this build. The change proposed here by @isuruf should fix this. Already it is apparent that the configuration issues that were seen on Travis CI are going away after this change. Thanks @isuruf.